docs/common.rst

```diff
@@ -92,74 +94,74 @@ An example of the begining of a file, including the `File Identification Stamp`_
   This is a convention often used by the CCHDO to record when changes were made to files
 
 .. warning::
-  Comments may contain UTF-8 encoded code points above U+007F, especially in proper names that may be present with data citation information.
-  If writing your own WHP-exchange reader, ensure that it can handle code points above U+007F or have it skip comment lines without trying to read them.
+  If writing your own WHP-Exchange reader, ensure that it can handle code points above U+007F or have it skip comment lines without trying to read them.
 
 .. _parameter and unit lines:
 
 Parameter and Unit Lines
 -------------------------
-.. warning::
-  There are additional headers specific to CTD WHP-exchange files.
+.. note::
+  There are additional headers specific to CTD WHP-Exchange files that come before parameter and unit lines.
   See the :ref:`CTD Specific Headers` section for details on these additional headers.
 
 After any format specific headers, the parameter and unit lines are next.
-The parameter names are first, units are second.
+The parameter names are first and unit lines are second.
 
-Parameter names are :unicode_info:`,` seperated values that define the columns the exchange file will contain.
-The names must be unique, capitalized, contain no empty fields, and not end with a trailing comma.
-The parameter names must contain only code points in the range U+0021 to U+007E except a :unicode_info:`,`.
-A trailing comma, or a comma that occurs at the end of the line with nothing else after it, MUST NOT be included on the parameter line.
-Certain parameter names, or parameter combinations, are required to be present.
-See the respective sections on :ref:`bottle required headers` and :ref:`CTD required headers` for information specific to each format.
+ * Parameter names are :unicode_info:`,` separated values that define the columns the exchange file will contain.
+ * The names MUST be unique, capitalized, contain no empty fields (each parameter must have a name), and not end with a trailing comma.
+ * The parameter names MUST contain only code points in the range U+0021 to U+007E excluding the lowercase range U+0061 to U+007A and excluding a :unicode_info:`,`.  **(These code points contain punctuation marks, numbers, brackets, basic math symbols, @, ^, `, ~, |, \, _.  Can I include all these marks for a parameter name in this code point range?  Including quotes?)**
```

Comment:
The answer is "maybe", but we should probably limit things more... there is no current defined parameter name convention, just an apparent one (all caps, underscore separated).